app/javascript/components/navbar/NavbarStyle.ts

```diff
@@ -0,0 +1,132 @@
+import styled from "styled-components";
+
+interface IModalProps {
+  open: boolean;
+}
+
+export const Wrapper = styled.div`
+  margin-top: -2rem;
+  height: 80px;
+  background-color: white;
+`;
+
+export const Container = styled.div`
+  display: flex;
+  max-width: 1200px;
+  height: 100%;
+  padding-left: 30px;
+  padding-right: 30px;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+export const Logo = styled.img`
+  margin-top: auto;
+  margin-bottom: auto;
+  padding-right: 160px;
+  cursor: pointer;
+  @media (max-width: 1100px) {
+    padding-right: 0px;
+  }
+  height: 30px;
+`;
+
+export const NavGroup = styled.div`
+  display: flex;
+  @media (max-width: 1100px) {
+    display: none;
+  }
+`;
+
+interface INavProps {
+  selected: boolean;
+}
+export const Nav = styled.div`
+  display: flex;
+  align-items: center;
+  height: 80px;
+  font-family: Poppins, sans-serif;
+  font-size: 16px;
+  padding-top: 6px;
+  color: #222326;
+  margin-right: 40px;
+  text-transform: uppercase;
+  user-select: none;
+  cursor: pointer;
+
+  ${(props: Partial<INavProps>) =>
+    props.selected === true &&
+    `
```

Comment:
I'm confused, what does this do?

Reply:
Allows us to pass the `selected` prop into `<Nav>` components like so :

```tsx
<Nav selected={props.navbarSelection === NavbarSelection.Channels}>
        {locale.navbar.channels}
      </Nav>
```

When `selected === true` then we render the purple highlight underneath the nav